ios/Exponent/Versioned/Core/Api/Components/Svg/Utils/RNSVGPathMeasure.m

```diff
@@ -20,7 +20,7 @@
 /**
  * returns the distance between two points
  */
-CGFloat distance(CGPoint p1, CGPoint p2)
+static CGFloat distance(CGPoint p1, CGPoint p2)
```

Comment:
Just curious, why is this change required?

Reply:
And why is there no Android counterpart?

Reply:
PR adding those changes is here:
* https://github.com/react-native-svg/react-native-svg/pull/1508